x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -169,19 +171,34 @@ async function claimAvailableTasks(opts: TaskClaimerOpts): Promise<ClaimOwnershi
   const now = new Date();
   const taskUpdates: PartialConcreteTaskInstance[] = [];
   for (const task of tasksToRun) {
-    taskUpdates.push({
-      id: task.id,
-      version: task.version,
-      scheduledAt:
-        task.retryAt != null && new Date(task.retryAt).getTime() < Date.now()
-          ? task.retryAt
-          : task.runAt,
-      status: TaskStatus.Running,
-      startedAt: now,
-      attempts: task.attempts + 1,
-      retryAt: getRetryAt(task, definitions.get(task.taskType)) ?? null,
-      ownerId: taskStore.taskManagerId,
-    });
+    // special check for invalid schedule interval
+    let intervalValidated = true;
+    if (task.schedule && task.schedule.interval) {
+      try {
+        parseIntervalAsSecond(task.schedule.interval);
+      } catch (err) {
+        intervalValidated = false;
```

Comment:
should we try either removing a `PT` prefix, or parsing with moment and then getting it's integer value?

Reply:
We can do that...and overwrite what's in the schedule interval?